### ISSUE TITLE
Plots with large numbers invisible

### ISSUE THREAD

**Issue by 2xB** [pyqtgraph/pyqtgraph#1011]
### Short description
If a plot contains an array with large numbers (e.g. 1e27), it is not drawn at all.

### Code to reproduce

```python
import numpy as np
import sys
import pyqtgraph as pg

app = pg.mkQApp()


x = [1,2,3,4]
y = [1,2,3,1e27]

pg.plot(x, y)


if (sys.flags.interactive != 1) or not hasattr(QtCore, 'PYQT_VERSION'):
    app.exec_()
```

### Expected behavior
A plot should be drawn.

### Real behavior
No plot is drawn.

### Tested environment(s)

 * PyQtGraph version: develop commit ad7453c7818eddb3b18354c65370f416cb668b29
 * Qt Python binding: PyQt5 5.13.0 Qt 5.13.0
 * Python version: 3.7.3
 * NumPy version: 1.16.4
 * Operating system: Manjaro
 * Installation method: pyqtgraph: Run in git repository; Dependencies: pip

**Comment by jongra** [comment]
same problem observed for `np.nan` values on windows system.

**Comment by 2xB** [comment]
@jongra Could you please post an example and screenshot? I tried the following on Windows and it looks fine.

```python3
import numpy as np
import sys
import pyqtgraph as pg

app = pg.mkQApp()

x = [1,2,3,4]
y = [1,2,3,np.nan]

pg.plot(x, y)

if (sys.flags.interactive != 1) or not hasattr(QtCore, 'PYQT_VERSION'):
    app.exec_()
```

**Comment by jongra** [comment]
Hi @2xB ,
thank you for your reply.
I just copied your example, but on my system (Win10 Education 64bit, i3-2120, Dell U2412M onboard graphics, python version 3.7.4, pygraph v. 0.10.0, PyQt5 v. 5.13.1, numpy v. 1.17.2) the plot is empty (cf. screenshot) and python doesn't raise any errors.
If I replace the `np.nan` with a `4`, it works fine and the graph shows a nice line plot.

![screen_1](https://user-images.githubusercontent.com/12088604/66753620-81b1e280-ee94-11e9-871c-979f0713526d.PNG)

**Comment by 2xB** [comment]
@jongra Interesting! My Windows system does not show this. Environment:

 * PyQtGraph version: both current develop and 0.10.0
 * Qt Python binding: PyQt5 5.12.2 Qt 5.12.3
 * Python version: 3.6.5
 * NumPy version: 1.14.3
 * Operating system: Windows 10
 * Installation method: Git, Anaconda

**Comment by jongra** [comment]
@2xB , yes indeed.
I've tried it today on a different windows machine with the same specs and it works fine. 
Seems to be a collision only on the first machine.
It would be very interesting for me to find the cause of this problem, since I cannot move to the second machine for production.
I would be grateful for any advice on how to debug the problem further.

**Comment by j9ac9k** [comment]
@jongra I'm not sure what can cause this discrepancy, if I was to start guessing, I would look at graphics drivers and test with a bunch of different Qt bindings (pyqt5 5.12.0 - 5.13.1, and same with pyside2).  Not that it should matter, but would it be possible to swap monitors between your two systems?

If you can come up with a test that detects this issue and fails on one machine, but passes on another, we would welcome that as a PR (as we would be able to test for this condition).

@2xB may have a further idea of what else to try;

**Comment by 2xB** [comment]
@jongra Thank you, that will be helpful! First, can you test if this still occurs when installing the newest version from the `develop` branch, so we do not track a bug that was already fixed? This can be done via

`pip install git+https://github.com/pyqtgraph/pyqtgraph@develop`

.

**Comment by 2xB** [comment]
I could reproduce the error, it occured after updating to PyQt5 5.13.1.

**Comment by jongra** [comment]
Yes, the error occurs only in the newest PyQt5 version 5.13.1. Version 5.13.0 works.
For me this solves the problem by now. Thank you all.

**Comment by j9ac9k** [comment]
This would likely be worth posting to the riverbank computing mail list.

**Comment by j9ac9k** [comment]
This should be fixed by #1140
